checkstyle.go

```diff
@@ -4,7 +4,8 @@ import (
 	"encoding/xml"
 	"fmt"
 
-	"gopkg.in/alecthomas/kingpin.v3-unstable"
+	"github.com/alecthomas/gometalinter/issues"
```

Comment:
Unfortunately, due to limitations of gopkg.in, we can't split into sub-packages. This will have to be moved back.

Reply:
Do you mean https://github.com/niemeyer/gopkg/issues/9? I think that can be fixed with https://golang.org/doc/go1.4#canonicalimports.The import can be changed to `gopkg.in/alecthomas/gometalinter.v1/issues` and the local code checked out or symlinked at that path. I think that would fix the problem

Reply:
Changing to gopkg.in imports then means that users can not `go get` the github source (which many people do).

It also makes development, and thus external contributions, more difficult:

- You can't just `go get` the github source and run it.
- If you do `go get` the github source, modify the issues subfolder, your changes will not be reflected. This will be confusing.
- You have to set up symlinks, or check out this repo in `$GOPATH/src/gopkg.in/alecthomas/gometalinter.v?`.

The extra burden to contributors (and users) outweighs the benefits.

Reply:
Don't get me wrong, it sucks, but there is no solution that doesn't suck even more (that I'm aware of).

Reply:
It does suck. I think the development/contributor issues are pretty easy to deal with, by either using a containerized development environment, or creating a symlink (which is trivial). However it's your call, so I have removed the extra package.

I hope that `go get` eventually adopts some of the features of `dep` so that we don't need `gopkg.in` anymore and we can actually have multiple packages...